web-api/tests/document_candidates.rs

```diff
@@ -71,65 +77,81 @@ async fn set(client: &Client, url: &Url, ids: impl IntoIterator<Item = &str>) ->
 
 #[test]
 fn test_candidates_all() {
-    test_app::<Ingestion, _>(UNCHANGED_CONFIG, |client, url, _| async move {
-        assert!(get(&client, &url).await?.ids().is_empty());
-        ingest(&client, &url).await?;
-        assert_eq!(get(&client, &url).await?.ids(), ["d1", "d2", "d3"].into());
-        set(&client, &url, ["d1", "d2", "d3"]).await?;
-        assert_eq!(get(&client, &url).await?.ids(), ["d1", "d2", "d3"].into());
-        Ok(())
-    });
+    test_app::<Ingestion, _>(
```

Comment:
all the changes in the integration tests, except for the new test for the filter, are just indention formatting changes due to the new argument in `test_app()`.